lib/std/os/windows.zig

```diff
@@ -2824,6 +2824,7 @@ pub const STD_OUTPUT_HANDLE = maxInt(DWORD) - 11 + 1;
 /// The standard error device. Initially, this is the active console screen buffer, CONOUT$.
 pub const STD_ERROR_HANDLE = maxInt(DWORD) - 12 + 1;
 
+/// Deprecated; use `std.builtin.CallingConvention.winapi` instead.
 pub const WINAPI: std.builtin.CallingConvention = .winapi;
```

Comment:
cc @squeek502 any objections?

Reply:
Don't have a strong opinion either way, but the alias `WINAPI` here seems roughly in line with other aliases in this file (e.g. `WINAPI` is just a `#define` like `BYTE`, `BOOL`, etc), so I'm not sure there's much of a reason to deprecate/remove it.

Reply:
The reason I don't like this one in particular is that people are using it in places they shouldn't be, e.g. here (which this PR fixes):

https://github.com/ziglang/zig/blob/ab89af3d397291f0126ad17d3787a51284457884/lib/std/start.zig#L171-L175

`callconv(.winapi)` is just clearly more readable for stuff like this.

Reply:
Makes sense, removing it seems totally fine as well.